api/src/test/java/com/epam/pipeline/controller/user/UserControllerTest.java

```diff
@@ -0,0 +1,504 @@
+    private static final String USERS_INFO_URL = USERS_URL + "/info";
+    private static final String GROUP_URL = SERVLET_PATH + "/group";
+    private static final String GROUP_FIND_URL = GROUP_URL + "/find";
+    private static final String GROUP_NAME_URL = GROUP_URL + "/%s";
+    private static final String GROUP_NAME_BLOCK_URL = GROUP_NAME_URL + "/block";
+    private static final String GROUPS_BLOCK_URL = SERVLET_PATH + "/groups/block";
+    private static final String ROUTE_URL = SERVLET_PATH + "/route";
+
+    private static final String EXPIRATION = "expiration";
+    private static final String NAME = "name";
+    private static final String PREFIX = "prefix";
+    private static final String BLOCK_STATUS = "blockStatus";
+    private static final String ROLE_IDS = "roleIds";
+    private static final String USER_NAME = "userName";
+    private static final String GROUP = "group";
+    private static final String GROUP_NAME = "groupName";
+    private static final String URL = "url";
+    private static final String TYPE = "type";
+    private final String redirectCookie =
+            "\"<html><body><script>window.location.href = \\\"TEST\\\"</script></body></html>\"";
+    private final String redirectForm =
+            "\"<html>\\n"
+            + "<body>\\n"
+            + "<form id=\\\"form\\\" method=\\\"post\\\" action=\\\"TEST\\\">\\n"
+            + " <input type=\\\"hidden\\\" name=\\\"bearer\\\" value=\\\"TEST\\\" />\\n"
+            + "</form>\\n"
+            + "<script>\\n" + "document.getElementById('form').submit()\\n"
+            + "</script>\\n"
+            + "</body>\\n"
+            + "</html>\"";
+    private static final String TEXT_HTML_UTF8_CONTENT_TYPE = "text/html;charset=UTF-8";
+
+    private final JwtRawToken token = SecurityCreatorUtils.getJwtRawToken();
+    private final PipelineUser pipelineUser = UserCreatorUtils.getPipelineUser();
+    private final PipelineUserVO pipelineUserVO = UserCreatorUtils.getPipelineUserVO();
+    private final UserInfo userInfo = UserCreatorUtils.getUserInfo(UserCreatorUtils.getPipelineUser());
+    private final CustomControl customControl = UserCreatorUtils.getCustomControl();
+    private final PipelineUserExportVO userExportVO = UserCreatorUtils.getPipelineUserExportVO();
+    private final GroupStatus groupStatus = UserCreatorUtils.getGroupStatus();
+
+    private final List<PipelineUser> pipelineUserList = Collections.singletonList(pipelineUser);
+    private final List<UserInfo> userInfoList = Collections.singletonList(userInfo);
+    private final List<CustomControl> customControlList = Collections.singletonList(customControl);
+    private final List<GroupStatus> groupStatusList = Collections.singletonList(groupStatus);
+
+    @Autowired
+    private UserApiService mockUserApiService;
+
+    @Autowired
+    private AuthManager mockAuthManager;
+
+    @Test
+    @WithMockUser
+    public void shouldGetSettings() {
```

Comment:
I see that in controller the method is called `getSettings` but it seems misleading. Probably we can call the test method `shouldGenerateToken`. Please also rename all test methods for the same controller method.